src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -443,9 +420,8 @@ public function finalizePackage()
      *
      * @param array $question
```

Comment:
PHPDoc for param $question can be removed as it does not contain any extra information, all we have in method declaration.